src/activemasternode.h

```diff
@@ -25,7 +25,7 @@ extern CActiveLegacyMasternodeManager legacyActiveMasternodeManager;
 class CActiveMasternodeInfo {
 public:
     // Keys for the active Masternode
-    CPubKey pubKeyMasternode;
+    CKeyID pubKeyIDMasternode;
```

Comment:
Should drop `pub` prefix i.e. s/`pubKeyIDMasternode`/`keyIDMasternode`/. Same for all other new `CKeyID`s.

Reply:
Done in separate commits. I'll probably go crazy because of this when doing the rebase of DIP3 :see_no_evil: :D